packages/components/atoms/f-form-field/src/assets/scss/common.scss

```diff
@@ -23,14 +23,16 @@ $form-input-borderColour--invalid       : $color-support-error;
 
 $form-input-focus--boxShadow            : 0 0 0 2px $color-focus;
 
+$form-input-padding--side              : spacing(x2);
+
 $form-input-height                      : spacing(x6);
-$form-input-padding                     : spacing(x1.5) spacing(x2);
+$form-input-padding                     : 11px $form-input-padding--side 13px;
```

Comment:
I am not a fan at all of have random px values in our code, seems like these spacings should come from PIE?

Reply:
Yeh, agree these look more random – the way centring works within the form-field to get things looking centred, the vertical padding (which is usually 12px top and bottom) actually needs to be 11px and 13px so that it actually lines up with the icon properly.

This could definitely come from PIE eventually, but at the moment these values don't exist in design tokens to do this.

Reply:
Ok makes sense, just note to come back and look at this in future